<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashReportDialog</class>
 <widget class="QDialog" name="CrashReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Send crash report</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lTitleInfoBug">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Send crash report</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="type" stdset="0">
        <string notr="true">title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lDescInfoBug">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>You can help us to improve MEGA by sending this error report to the Helpdesk.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lDescribeBug">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Describe the issue</string>
       </property>
       <property name="semibold" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="teDescribeBug">
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="type" stdset="0">
        <string notr="true">mega</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="cbAttachLogs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Attach diagnostic log files to assist with debugging</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="type" stdset="0">
          <string notr="true">mega</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wNotificationBanner" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 8;</string>
       </property>
       <property name="type" stdset="0">
        <string notr="true">notification</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lInfoBanner">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">notification</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>219</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bSubmit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Send report</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="type" stdset="0">
         <string notr="true">primary</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">outline</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>CrashReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
